### ISSUE TITLE
Accessing files from private bucket

### ISSUE THREAD

**Issue by pljspahn** [ehossack/django-backblaze-b2#22]
I'm a little unclear on the config necessary for using a private bucket.

```
CACHES = {
    'default': {
        'BACKEND': 'django.core.cache.backends.dummy.DummyCache',
    },
    'django-backblaze-b2': {
        'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
    }
}

BACKBLAZE_CONFIG = {
    'application_key_id': 'my_id',
    'application_key': 'my_key',
    'bucket': 'my_bucket',

}
```

Model Field:
```
class MyModel(models.Model):
    ...
    backblaze_file = models.FileField(
        upload_to='uploads',
        storage=PublicStorage,
    )
```

This setup allows me to save/upload the file, but then when accessing it via local URL (mydomain.com/b2/uploads/my_image_file.jpg) I simply get `A server error occurred.  Please contact the administrator.` in the browser along with `TypeError: download_file_by_name() got an unexpected keyword argument 'download_dest'` in the terminal.

**Comment by ehossack** [comment]
I went to double check your settings and they look correct to me. I'm using a private bucket in production with no issues.

What version of the b2 sdk are you using? I'm in the process of updating the library for the latest python and SDK, but as of yet it's only tested up to `1.9`

**Comment by pljspahn** [comment]
b2sdk=1.14.0

It looks like you have dependencies setup to install b2sdk >=1.9.0

https://github.com/ehossack/django-backblaze-b2/blob/556777a74a23780bffde68296c3173fb5a7d5ccd/pyproject.toml#L46

If it's not been tested about 1.9, then I assume it's a matter of changing that to `b2sdk=1.9.0`

**Comment by ehossack** [comment]
This should be fixed in version 3.0

Dependabot has been enabled too so that any future breaking changes can be caught, and a minor version capping them released if necessary.
